bazel/README.md

```diff
@@ -364,7 +364,7 @@ for more details.
 
 ## Supported compiler versions
 
-We now require Clang >= 9 due to C++17 support and tcmalloc requirement. GCC >= 9 is also known to work.
+We now require Clang >= 14 due to C++20 support and tcmalloc requirement. GCC >= 11 is also known to work for C++20.
```

Comment:
```suggestion
We now require Clang >= 18 due to C++20 support and tcmalloc requirement. GCC >= 11 is also known to work for C++20.
```

i think we need latest compiler for some features, and anything other than what we are testing on is unlikely to be well supported - maybe even say - == 18 - for other compilers ymmv